readthedocs/proxito/middleware.py

```diff
@@ -199,6 +200,10 @@ def process_response(self, request, response):  # noqa
         * For custom domains, check the HSTS values on the Domain object.
           The domain object should be saved already in request.domain.
         """
+        # Reset URLconf for this thread
+        # to the original one.
```

Comment:
why not put the comment in just one line? :smile:

Reply:
I just like short lines to have more space in my monitor :man_shrugging: